lib/dashing/app.rb

```diff
@@ -120,16 +119,6 @@ def protected!
   end
 end
 
-Thin::Server.class_eval do
-  def stop_with_connection_closing
-    Sinatra::Application.settings.connections.dup.each(&:close)
-    stop_without_connection_closing
-  end
-
-  alias_method :stop_without_connection_closing, :stop
-  alias_method :stop, :stop_with_connection_closing
-end
-
```

Comment:
I was not really sure about this :-\ It appears to monkey-patch Thin to close all open connections during shutdown. I don't think there needs to be an equivalent for Puma, because it already manages those connections and has a graceful shutdown path. Perhaps Thin would refuse to shut down unless all connections were closed?